packages/components/src/dialog/dialog.tsx

```diff
@@ -15,6 +15,15 @@ interface Props {
   press?: 'normal' | 'long'
 }
 
+const Wrapper = styled(Stack, {
```

Comment:
Don't find this ideal, but consider this as quick fix, otherwise Dialog content didn't show up in App as it's pushed below fold because of 100vh in app.css.